README.md

```diff
@@ -75,9 +75,21 @@ instead.
 To run the project, type:
 
 ```
-$ dune exec prjt_mi_recolor
+$ dune exec prjt_mi_recolor file.txt
 ```
 
+The file must be in the following format:
+```
+l w
+n11 n12 n13 ... n1l
+n21 n22 ... ... ...
+n31 ... ... ... ...
+... ... ... ... ...
+nw1 nw2 nw3 ... nwl
```

Comment:
Perhaps we are not obliged to specify the coordinates, as is implicit?

Reply:
https://github.com/GabinDDL/Math-Info-L3-Project/pull/15/files#r1524331279

Reply:
> Perhaps we are not obliged to specify the coordinates, as is implicit?

I'm having a bit of trouble understanding, basically I specify for the 1st line the number of rows and columns, and then how the table is and the n represents the color and only the color of the coordinates where it is, for example n11 if the box of coordinates 1 1 must be color 2, will be equal to 2